spec/calculate_spec.rb

```diff
@@ -17,58 +16,55 @@
 
       expect(Calculate.format_date).to eql(today_date_formatted)
     end
-    it 'if date passed in, return passed in value', :focus do
-      expect(Calculate.format_date("123456")).to eql("123456")
-    end
-    it 'if date passed in, raises error if length does not equal 6' do
-      expect{Calculate.format_date("31420")}.to raise_error(ArgumentError)
-    end
-    it 'if date passed in, raises error if contains characters besides numbers' do
-      expect{Calculate.format_date("453f34")}.to raise_error(ArgumentError)
-    end
+
+      context 'when date passed in' do
+
+        it 'return passed in value'  do
+          expect(Calculate.format_date("123456")).to eql("123456")
+        end
+
+        context 'when length does not equal 6' do
+          it 'raises error' do
+            expect{Calculate.format_date("31420")}.to raise_error(ArgumentError)
+          end
+        end
+
+        context 'when containing characters besides numbers' do
+          it 'raises error ' do
+            expect{Calculate.format_date("453f34")}.to raise_error(ArgumentError)
+          end
+        end
+      end
   end
 
   describe '::offsets' do
-
     it 'returns hash containing last four digits of date squared' do
```

Comment:
I might have the first test be something obvious that someone that is reviewing the code could easily do in their head:

010004 -> "0 0 1 6"

And then the duplicated bit could move up into a context block and then the different part is more descriptive: 

```
it 'a trivial number' do
...
it 'a very large number' do
```